### DISCUSSION TITLE
Page width setting

### DISCUSSION THREAD

**Discussion by exen904** [hugo-sid/hugo-blog-awesome#99] [Q&A]
Not sure if Im alone with this request, but a page width setting, in the best case in the configuration.toml would be great. Its a bit narrow especially on ultrawide displays and I dont really want to mess with the whole theme just for adjusting that.

**Comment by hugo-sid** [comment]
Hi @exen904, you don't have to 'mess with the whole theme just for adjusting' page width.

Take a look at this relevant discussion : https://github.com/hugo-sid/hugo-blog-awesome/discussions/46
